aiotg/bot.py

```diff
@@ -78,6 +82,42 @@ def no_handle(mt):
             )
             self._process_updates(updates)
 
+    def webhook_loop(self, loop):
+        """
+        Starts aiohttp web server.
+        """
+        app = web.Application(loop=loop)
+
+        async def handle(request):
+            update = await request.json()
+            self._process_update(update)
+            return web.Response()
+        app.router.add_route('POST', urlparse(self.webhook_url).path, handle)
+        web.run_app(app)
+
+    def set_webhook(self, url, **options):
+        """
+        Register you webhook url for Telegram service.
+
+        :Example:
+
+        >>> bot.set_webhook(url='https://yourserver.com/webhook/<token>')
+
+        Additional documentation on https://core.telegram.org/bots/api#setwebhook
+        """
+        self.webhook_url = url
+        return asyncio.get_event_loop().run_until_complete(self.api_call(
```

Comment:
I'm not sure if it's good idea to make set_webhook blocking. I mean in most cases it's called only once but in general, we should make async operations explicit and don't hide them behind run_until_complete().

So what if we make set_webhook() explicitly async, **but** let users pas optional parameter *webhook* to run() so they don't have to manually await set_webhook()? run() already uses run_until_complete() so it should be fine.